components/com_config/tmpl/config/default.php

```diff
@@ -34,7 +34,7 @@
         </h1>
     </div>
 <?php endif; ?>
-<form action="<?php echo Route::_('index.php?option=com_config'); ?>" id="application-form" method="post" name="adminForm" class="form-validate">
+<form action="<?php echo Route::_('index.php'); ?>" id="application-form" method="post" name="adminForm" class="form-validate">
```

Comment:
```suggestion
<form action="<?php echo Route::_('index.php'); ?>" method="post" name="adminForm" id="application-form" class="form-validate">
```

action/method/name/id/class